libs/remix-ui/static-analyser/src/lib/ErrorRenderer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react' //eslint-disable-line
+
+interface ErrorRendererProps {
+  message: any;
+  opt: any,
+  warningErrors: any
+}
+
+const ErrorRenderer = ({ message, opt }: ErrorRendererProps) => {
+  const [, setError] = useState(
+    {
+      row: null,
+      column: null,
+      text: null,
+      type: null,
+      errFile: null
+    }
+  )
+  const getPositionDetails = (msg: any) => {
+    const result = { } as any
```

Comment:
We can give result a better type. (Record<string, int | string>)

Reply:
`type` added